1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,31 @@
 function highlightWords(paragraph, colours) {
-  // Write your code here...
+  const container = document.getElementById("content");
+  const select = document.createElement("select");
+  const option = document.createElement("option");
+  option.innerText = "Select colour";
+  select.appendChild(option).disabled = true;
+  container.appendChild(select);
+  const para = document.createElement("p");
+  container.appendChild(para);
+
+  colours.forEach((colour) => {
+    const opt = document.createElement("option");
+    opt.innerText = colour;
+    select.appendChild(opt);
+  });
+
+  paragraph.split(" ").forEach((word) => {
+    const span = document.createElement("span");
+    span.innerText = word + " ";
+    para.appendChild(span);
+
+    span.addEventListener("click", () => {
```

Comment:
Very good. Can you think of a way to have a seperate function to handle the onClick event?

e.g. span.addEventListener('click', updateColour);

Do yo know how updateColour would knwo which soan had been clicked?